lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'menu'
+require_relative 'sendsms'
+
+class TakeAway
+
+  attr_reader :menu, :order
+
+  def initialize menu = Menu.new
+    @menu = menu
+    @order = Hash.new(0)
+    # @basket = Array.new
+  end
+
+  def menu_list
+    @menu.show_menu
+  end
+
+  def add_meal item, quantity = 1
+    fail 'not in the menu' unless menu.dishes.include?(item)
+    order[item] += quantity
+    "#{quantity} #{item} has been added to your order"
+  end
+
+  def delete item, quantity = 1
+    fail 'dish not in the menu' unless menu.dishes.include?(item)
+    order[item] -= quantity
+  end
+
+  def order_summary
+    summary = order.map{|item,quantity|"#{item} x #{quantity} = £#{(menu.dishes[item]*quantity)}"}.join(', ')
```

Comment:
Metrics/LineLength: Line is too long. [109/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)